src/components/Dropdown/Dropdown.styles.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from "styled-components";
+
+export const NavItem = styled.li`
+  position: relative;
+  transition: 0.3s;
+  line-height: 1.5rem;
+  list-style: none;
+  border-left: thin solid transparent;
+  margin: 0;
+
+  a {
+    display: block;
+    text-decoration: none;
+    width: 100%;
+    font-weight: normal;
+    font-style: none;
+  }
+
+  @media screen and (max-width: 1000px) {
+    width: 100%;
+
+    &:hover {
+      padding-left: 0.5rem;
+      border-left: thin solid ${({ theme }) => theme.colors.black};
+    }
+  }
+`;
+
+export const ItemContent = styled.div`
+  display: flex;
+  gap: 0.25rem;
+  align-items: center;
+  transition: 0.3s;
+
+  a {
+    display: block;
+    text-decoration: none;
+    width: 100%;
+    font-weight: normal;
+    font-style: none;
+    text-wrap: nowrap;
+  }
+
+  &:hover {
+    opacity: 0.6;
+  }
+`;
+
+export const Wrapper = styled.div`
+  padding-top: 1rem;
+
+  position: absolute;
+  background-color: transparent;
+  width: 100%;
+  height: 100%;
+`;
+
+export const ChildrenWrapper = styled.ul`
+  margin: 0;
+  position: absolute;
+  border-radius: 4px;
+  width: fit-content;
```

Comment:
I guess we can add some min-width or something like that to avoid break lines like that:
![image](https://github.com/user-attachments/assets/6020fe34-d554-4eee-8e02-e82ca4b96b9c)